src/composables/useVeBAL.ts

```diff
@@ -18,7 +23,7 @@ const showRedirectModal = ref(false);
  * COMPUTED
  */
 export const isVeBalSupported = computed(
-  () => isMainnet.value || isKovan.value
+  () => isMainnet.value || isKovan.value || isArbitrum.value || isPolygon.value
```

Comment:
I've added the other networks instead of just a blanket true in case we need to fine tune it later on

Reply:
I thought users can't vote on gauges or lock veBAL on L2? This value is used for those pages.

Reply:
yeah damn, I'll see another way

Reply:
@timjrobinson fixed :)